Sources/OpenAPIRuntime/Conversion/CodableExtensions.swift

```diff
@@ -189,7 +189,7 @@ extension Encoder {
 }
 
 /// A freeform String coding key for decoding undocumented values.
-private struct StringKey: CodingKey, Hashable, Comparable {
+struct StringKey: CodingKey, Hashable, Comparable {
```

Comment:
Should this be `@_spi`?

Reply:
Same as above, internal.

Reply:
Hmm actually this isn't needed anymore, let me revert.

Reply:
https://github.com/apple/swift-openapi-runtime/pull/66/commits/5fa5ab097dc875756a05ae09437a07785decadb8